extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1766,6 +1770,22 @@ export class AmpA4A extends AMP.BaseElement {
   isVerifiedAmpCreative() {
     return this.isVerifiedAmpCreative_;
   }
+
+
+  /**
+   * Adds single pass experiment IDs if the javascript binary has
+   * "singlePassType" mode.
+   */
+  maybeAddSinglePassExperiment() {
+    const type = getMode().singlePassType;
+    if (type === 'sp') {
+      addExperimentIdToElement(SINGLE_PASS_EXPERIMENT_IDS.SINGLE_PASS,
+          this.element);
+    } else if (type === 'mp') {
+      addExperimentIdToElement(SINGLE_PASS_EXPERIMENT_IDS.MULTI_PASS,
+          this.element);
+    }
```

Comment:
What do normal production binaries return here?  Do we expect `sp` and `mp` equally often?

Reply:
normal binaries would usually not even have the field in AMP_CONFIG. i should adjust my tests for that.

yes we expect sp/mp equally often